dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -668,61 +654,40 @@ public static void saveMultiTree(MultiTree o, long languageId) throws DotSecurit
      * @throws DotSecurityException 
      *            
      */
-    private static void updateVersionTs(String id) throws DotDataException {
-        updateVersionTs(id, APILocator.getLanguageAPI().getDefaultLanguage().getId());    
-    }
-	
-    /**
-     * Update a HTML Page Version Info Timestamp given a HTMLPage Identifier and a Language Id.
-     * If a MultiTree Object has been added or deleted from this page,
-     * its version_ts value needs to be updated so it can be included
-     * in future Push Publishing tasks
-     * 
-     * @param id The HTMLPage Identifier to pass in 
-     * @throws DotContentletStateException
-     * @throws DotDataException 
-     * @throws DotSecurityException 
-     *            
-     */
-	private static void updateVersionTs(String id, Long languageId) throws DotDataException {
-	    Identifier ident = APILocator.getIdentifierAPI().find(id);
-        ContentletVersionInfo versionInfo = APILocator.getVersionableAPI()
-                .getContentletVersionInfo(ident.getId(), languageId);
-        versionInfo.setVersionTs(new Date());
-        APILocator.getVersionableAPI().saveContentletVersionInfo(
-                versionInfo);
+	private static void updateHTMLPageVersionTS(String id) throws DotDataException, DotSecurityException {
+		Identifier identifier = APILocator.getIdentifierAPI().find(id);
+		List<Contentlet> allContentletVersions = APILocator.getContentletAPI()
+			.findAllVersions(identifier, APILocator.getUserAPI().getSystemUser(), false);
```

Comment:
This isn't the right method - this pulls all old versions of a piece of content which can be in the 100's .  We need to pull a list of ContentletVersionInfos for a given piece of content